src/main/scala/devices/debug/Periphery.scala

```diff
@@ -74,6 +75,11 @@ trait HasPeripheryDebug { this: BaseSubsystem =>
 
   val debugCustomXbarOpt = p(DebugModuleKey).map(params => LazyModule( new DebugCustomXbar(outputRequiresInput = false)))
   val apbDebugNodeOpt = p(ExportDebug).apb.option(APBMasterNode(Seq(APBMasterPortParameters(Seq(APBMasterParameters("debugAPB"))))))
+  val debugTLDomainOpt = p(DebugModuleKey).map { _ =>
+    val domain = ClockSinkNode(Seq(ClockSinkParameters()))
+    domain := tlbus.fixedClockNode
+    domain
+  }
   val debugOpt = p(DebugModuleKey).map { params =>
     val debug = LazyModule(new TLDebugModule(tlbus.beatBytes))
```

Comment:
I could alternatively add a ClockSink to the TLDebugModule itself. I choose this because it was less invasive.

Reply:
Yeah I have been hesitant to add clock nodes to any of the device modules because it makes them even harder to instantiate "stand-alone" and I kind of like the separation of concerns offered by the wrappers. It's just unfortunate that the wrapping pattern depends on all IOs being diplomatic, which is often not a problem but is for these older peripherals like the debug and clint rtc tick. Anyways I think this approach is fine for now.